### PR TITLE
Fix bug where wrong function name was used

### PR DESCRIPTION
**Pull request by oyvindeide**
**Issue**
Resolves #my_issue

**Approach**
_Short description of the approach_

## Pre review checklist

- [x] Added appropriate release note label
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

File: tests/unit_tests/gui/tools/test_case_tool.py [hunk 1]
```diff
@@ -27,3 +27,30 @@ def test_case_tool_init_prior(qtbot):
         list(storage.getStateMap())
         == [RealizationStateEnum.STATE_INITIALIZED] * ert.getEnsembleSize()
     )
+
+
+@pytest.mark.usefixtures("copy_poly_case")
+def test_that_case_tool_can_copy_case_state(qtbot):
```
Comment (pinkwah) [inline tests/unit_tests/gui/tools/test_case_tool.py:33]: 1. I think this fairly heavy test for what it's supposed to do. Ie, we instantiate the whole ERT machinery to see if the button works.
2. The test doesn't necessarily test what we want. Only that the ensemble has switched from `UNDEFINED` to `INITIALIZED` which doesn't mean that any data has been copied correctly.

I have no hard opinions on this, so we can merge if we can't come up with a better test.

Comment (oyvindeide) [reply tests/unit_tests/gui/tools/test_case_tool.py:33]: 1. Agreed, but it is not terribly slow (400 ms) on my machine
1. Can make it more explicit if you want (testing value equality), but that requires even more boiler plate

I am hesitant to do more `mock`ing, as I might mock away what this test is here to check regression against. Do you have suggestions for improvement?

Comment (oyvindeide) [reply tests/unit_tests/gui/tools/test_case_tool.py:35]: I updated the test, checking that the parameters were copied over.

Comment (oyvindeide) [reply tests/unit_tests/gui/tools/test_case_tool.py:36]: Updated again with simpler case that runs faster (130 ms on my machine)

**Pull request merged**